scripts/release.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -x
+echo "Running release.sh"
+
+echo "Migrating database"
+python manage.py migrate --noinput
```

Comment:
I moved this from the entry point to the release phase, because:

> If a release phase task fails, the new release is not deployed, leaving your current release unaffected.

https://devcenter.heroku.com/articles/release-phase

Migrations are explicitly named as good candidates for the release phase for this reason.